README.md

```diff
@@ -78,6 +78,7 @@
 - [dAppTemplate_MEAN](https://github.com/CityOfZion/dAppTemplate_MEAN) - Interface for communicating with NEO nodes, to help with development of a dApp.
 - [Java smart contracts](https://github.com/neo-project/examples-java) - Examples of smart contracts written in Java from NEO core team.
 - [Kotlin smart contracts](https://github.com/neo-project/examples-kotlin) - Examples of smart contracts written in Kotlin from NEO core team.
+- [Community Code Snippets](https://github.com/nickazg/neo-snippets) - Categorised Examples of helpful code snippets for Smart Contracts
```

Comment:
Please put the links in alphabetical order to keep the list tidy :)

Reply:
And remove the capital characters from "Examples", "Smart" and "Contracts" for consistency with the rest of the list

Reply:
Lastly, remember to put a . at the end of the sentence ;)

Reply:
Nice idea to make a list like this! Few small changes and it should be good